packages/svelte/src/lib/components/SSR.svelte

```diff
@@ -1,9 +1,15 @@
-<script lang="ts">
+<script context="module" lang="ts">
   import type { Page } from '@inertiajs/core'
+  export type SSRProps = { id: string; initialPage: Page }
```

Comment:
No changes here. Exporting the types to use in createInertiaApp for the SSR instead of casting to any for better typing.